### ISSUE TITLE
talkback has moved to /app

### ISSUE THREAD

**Issue by mfonville** [opengapps/opengapps#14]
On Android 5.1 talkback has moved from /priv-app to /app.
This change has been incorporated into all versions, but has to be tested if it works on versions lower than 5.1.

**Comment by rapperskull** [comment]
I think that we should respect the Google factory images, even if it's all working fine. I think that on 5.0 talkback should be in /system/priv-app and Hangouts in /system/app.

**Comment by mfonville** [comment]
Well, if it is not necessary, it would be 'safer' for the system to have it in /system/app instead of /system/privapp, and I think that the result that they moved is, is probably because of newer/better insights and/or updates to the app(s) so that they don't require these root-privileges anymore.
Also, being able to have one location per app is more easy to (maintain in the) code and also ensures that upgrades go without any trouble (e.g. the gapps-remove.txt file does not only know about the paths of its own platform level)

**Comment by mfonville** [comment]
Talkback is confirmed to function correctely when put in /app on Android 5.0 and 4.4, so I close this issue, since it has been researched now.
